src/BuiltInTools/dotnet-watch/Program.cs

```diff
@@ -147,6 +147,11 @@ internal static RootCommand CreateRootCommand(Func<CommandLineOptions, Task<int>
             root.TreatUnmatchedTokensAsErrors = false;
             root.Handler = CommandHandler.Create((CommandLineOptions options, ParseResult parseResults) =>
             {
+                if (!string.IsNullOrEmpty(parseResults.ValueForOption<string>("-p")))
+                {
+                     reporter.Warn(Resources.Warning_ProjectAbbreviationDeprecated);
```

Comment:
What does this actually look like in the console output?  It looks like we're going to go with the following (in yellow text) for `dotnet run`:

> Warning NETSDKNNNN: The abbreviation of -p for --project is deprecated. Please use --project.

Where NNNN is replaced with an SDK error code, so you can better search for information about the error.

Reply:
Here's what this looks like now sans the SDK error code:

![image](https://user-images.githubusercontent.com/174281/122602404-9cecd200-d027-11eb-9866-b2cf34144089.png)

Have you decided on what the error code is?

Reply:
NETSDK1174 - https://github.com/dotnet/sdk/pull/18301